Documentation/ceph-object.md

```diff
@@ -105,6 +105,43 @@ To directly retrieve the secrets:
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep AccessKey | awk '{print $2}' | base64 --decode
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep SecretKey | awk '{print $2}' | base64 --decode
 ```
+## Create an Object Bucket Claim
+
+Using Object Bucket Claim(otherwise known as obc), a bucket will be provisioned from object store for the application. It can consume already existing buckets or else new bucket will be created. For this first a storage class need to be defined and then use it to create object bucket claim, all using prefered settings. See the [Object Bucket Claim CRD](ceph-object-bucket-claim-crd.md) for more detail on the settings available for a `CephObjectBucketClaim`.
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
```

Comment:
Need to mention that this storage class is for new bucket provisioning -- not existing buckets.

Reply:
Done